regressions/regressions.xml

```diff
@@ -42,5 +42,6 @@
         <test testName="test-1649" configFile="config/TEMPLATE_regressions.xml" />
         <test testName="test-1996" configFile="config/TEMPLATE_regressions.xml" />
         <test testName="test-2041" configFile="config/TEMPLATE_regressions_thrust.xml" />
+        <test testName="cuda_device_prop" configFile="config/TEMPLATE_regressions.xml" />

```

Comment:
this regression case is for bug fix in https://github.com/oneapi-src/SYCLomatic/pull/32